pom.xml

```diff
@@ -60,6 +60,11 @@
             <version>1.9</version>
         </dependency>
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
If you wanted to minimise dependencies, you could just use jackson-core, and pull the name out using the jackson token parser.